initialise/initialise.go

```diff
@@ -57,10 +64,23 @@ func (e *ExternalServiceList) GetConsumer(ctx context.Context, cfg *config.Confi
 	return
 }
 
+const base = 10
+const bitSize = 32
+
```

Comment:
Minor _nit-picky-code-nazi-comment_ its good practice/convention to define variable at constants at the top of the file. As general rule of thumb for go file structure.

> import
> vars/constants
> types
> methods/functions